### PR TITLE
#164797149 users can create article

### PR DESCRIPTION
**Pull request by emmsdan**
#### What does this PR do? 
 - allows users to create an article

#### Description of Task to be completed?
- [x] write tests for create article functionality
- [x] users should be able to create an article

#### How should this be manually tested?

* clone `https://github.com/andela/zinnia-ah-backend.git`
* pull from the remote branch to update the local branch
* checkout to  the branch = `ft-create-article-164797149 `
* run `npm install && install -g pg`
* run `cp .env.example .env`
* run `sequelize db:migrate`
* run npm test
* run npm run start:dev
* using postman visit: ***POST*** `http://localhost:3000/api/v1/article`,
 ```javascript
         title, description, body. [optional] tags, images
 ```
#### What are the relevant pivotal tracker stories?(if applicable)
#164797149

#### Screenshots
Screenshot 2019-04-03 at 7 08 16 PM

### CODE REVIEW

File: src/tests/db/controller/article.test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+let xAccessToken = '';
+
+describe('CREATE ARTICLE', () => {
+  it('should create user, to enable us use jwt token', (done) => {
+    chai.request(app)
+      .post('/api/v1/users')
+      .send(userRequestObject)
+      .end((err, res) => {
+        expect(res.status).to.equal(201);
```
Comment (Japhethca) [inline src/tests/db/controller/article.test.js:32]: let's test for the response data.

Comment (emmsdan) [reply src/tests/db/controller/article.test.js:32]: thanks, I have tested for the response data

File: src/db/models/Article.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const Article = sequelize.define('Article', {
+    articleId: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+      autoIncrement: false,
+      allowNull: false,
+      field: 'id',
+    },
+    userId: {
```
Comment (Japhethca) [inline src/db/models/Article.js]: let's rename this to `authorId`.

Comment (emmsdan) [reply src/db/models/Article.js]: ok, thanks, let me fix that.

File: src/db/models/Article.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+module.exports = (sequelize, DataTypes) => {
```
Comment (Japhethca) [inline src/db/models/Article.js]: we can export default here instead of `module.exports`

File: src/db/models/User.js [hunk 4]
```diff
@@ -2,6 +2,13 @@ import bcrypt from 'bcryptjs';
 
 module.exports = (sequelize, DataTypes) => {
```
Comment (Japhethca) [inline src/db/models/User.js]: let's use `export default` instead

**Review by Japhethca — COMMENTED** [review]
looks good!

**Review by Japhethca — APPROVED** [review]

**Pull request merged**